vb_toolbox/app.py

```diff
@@ -75,21 +75,28 @@ def create_parser():
                         help="""File containing the surface clusters. Cluster
                         with index 0 are expected to denote the medial brain
                         structures and will be ignored.""")
+                        
+    parser.add_argument('-t', '--tol', metavar='tolerance', type=float, nargs=1,
+                        default=["def_tol"], help="""Residual tolerance (stopping criterion) for LOBPCG. 
+                        Default value = sqrt(10e-18)*n, where n is the number of nodes per graph.""")
+                        
+    parser.add_argument('-mi', '--maxiter', metavar='max iterations', type=int, nargs=1, default=[50],
```

Comment:
I believe the default value should be a single element and not a list (default="def_tol", and default=50). Also for the tolerance criterion, I would list the accepted values in the help string)